DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -99,6 +103,22 @@ - (NSArray *)sections
     return _sections;
 }
 
+- (NSAttributedString *)attributedTextWithString:(NSString *)string
+{
+    if (string) {
+        return [[NSAttributedString alloc] initWithString:string];
+    }
+    return nil;
+}
+
+- (NSAttributedString *)attributedPlaceholderTextWithString:(NSString *)string
+{
+    if (string) {
+        return [[NSAttributedString alloc] initWithString:string];
+    }
+    return nil;
+}
```

Comment:
It looks like `attributedTextWithString` and `attributedPlaceholderTextWithString` are doing the same thing so you may be able to just create a single method.